### ISSUE TITLE
Replacing the controller

### ISSUE THREAD

**Issue by volton-svg** [gianlucarenzi/stalker#5]
Good afternoon! Is it possible to replace 32stm401rct6 with 32stm401ret6?

**Comment by reinauer** [comment]
Yes, the footprint is the same for both the STM32F401RCT6 and STM32F401RET6. Both microcontrollers come in the LQFP64 (64-pin) package, meaning they have the same physical dimensions, pin layout, and footprint. This makes them interchangeable in terms of hardware design, provided that the memory difference is accounted for in the application.
